TRACK_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thank you for this PR! :tada:
+
+When we review your submission we will look for the following.
+
+- [ ] Trailing whitespace and extraneous blank lines 
+- [ ] Indentation and code formatting issues
+- [ ] Spelling and grammatical errors
+- [ ] Code style
+- [ ] Use of non-standard libraries
+- [ ] Extraneous config files
+- [ ] Deviations from the specifications set out for the exercise in [x-common](https://github.com/exercism/x-common/tree/master/exercises)
```

Comment:
> This gets to a core question within Exercism. How much variance would we like to allow in exercise implementations?

Yeah, we definitely want to let languages follow as closely or loosely as they want to, in order to do what is best for the language.

I think we should consider finding language that will give this "flavor" to the bullet point